Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb_example.py

```diff
@@ -90,14 +95,21 @@ def meas():
 ##############################
 ##  Two-qubit RB execution  ##
 ##############################
-
+# create RB experiment from configuration and defined functions
 rb = TwoQubitRb(
-    config, bake_phased_xz, {"CZ": bake_cz}, prep, meas, verify_generation=False, interleaving_gate=None
-)  # create RB experiment from configuration and defined functions
+    config=config,
+    single_qubit_gate_generator=bake_phased_xz,
+    two_qubit_gate_generators={"CZ": bake_cz},  # can also provide e.g. "CNOT": bake_cnot
+    prep_func=prep,
+    measure_func=meas,
+    interleaving_gate=None,
```

Comment:
Did you check that it also works with an interleaved gate?
It would be nice to have an example with how to define this gate for cirq newbies like me

Reply:
I don't have an example yet, but I will add and test it in the future. I also don't know if it works- but it wont be trivial to test, so I will make a new pull-request for that later.